### ISSUE TITLE
Too many Enchantments can block ENCHANT.

### ISSUE THREAD

**Issue by MinerB4** [matthewperiut/Entris#1]
Having too many enchantments blocks the ENCHANT button.
Maybe have it to where ENCHANT is on the Left Side, or works when you press Enter/Return.

**Comment by matthewperiut** [comment]
You're right, I will work on implementing moving the button and making enter press the enchant button.

**Comment by GoldenFurredFox** [comment]
Maybe instead of moving the button, have a scrolling GUI?

**Comment by Diego13303** [comment]
On the topic of reworking the UI, having the ability to unselect enchantments would be pretty good. I think it´d also be good to make it more clear which enchantments do you have currently selected and how many can you select

**Comment by Ynaught** [comment]
> You're right, I will work on implementing moving the button and making enter press the enchant button.

I would rather a page button system:

`[Option #1]`
`[Option #2]`
`[Option #3]`
`[Option #4]`
`[Option #5]`
`[Option #6]`
 `[<]` 1/2 `[>]`
 
`[ENCHANT!]`

**Comment by matthewperiut** [comment]
smart, i think thats way better

**Comment by Ynaught** [comment]
> smart, i think thats way better

Yeah, it's faster than scrolling, compact, but allows for infinite enchants.

You could even make the "1"/2 a text box so you could manually type the number in if the player wishes. That being said, that would imply you got over 9 pages of enchants to go through for such a feature to even be convient in the first place. . . and personally speaking, I "doubt" such a circumstance would be that frequent of an issue to justify the effort for the textbox implementation.
